extension/intents/timer/timer.toml

```diff
@@ -11,40 +11,55 @@ phrase = "Set timer for 2 hours and 5 minutes"
 [timer.close]
 description = "Close timer"
 match = """
+<<<<<<< HEAD
```

Comment:
Looks like you added some conflict markers

Reply:
My bad. Will fix that now